manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -121,12 +121,15 @@ const toPrivateOnlySignTest = async () => {
 
     await privateWallet.initalWalletSync();
 
-    const initalPrivateBalance = await privateWallet.getPrivateBalance(assetId);
-    console.log("The inital private balance is: ", initalPrivateBalance.toString());
+    const initialPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+    console.log("The initial private balance is: ", initialPrivateBalance.toString());
 
-    const signResult = await privateWallet.toPrivateSend(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
+    const signResult = await privateWallet.toPrivateBuild(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
 
-    console.log("The result of the signing: ", signResult);
+    console.log("The result of the signing: ", JSON.stringify(signResult));
+
+    // remove first 3 bytes of the signResult
+    console.log("For xcm remote transact payload, please use: \"0x" + JSON.stringify(signResult.txs).slice(10));
```

Comment:
We could add a method to `privateWallet.ts` that does this. 

Like `toPrivateBuildXcm()`

I am cool with it as is too

Reply:
Ill do this in a separate PR and also add the private-transfer and to-public: https://github.com/Manta-Network/sdk/issues/70